packages/editor/src/components/blocks/radio/Radio.tsx

```diff
@@ -0,0 +1,88 @@
+import type { Prettify, Radio, AlignmentType } from '@axonivy/form-editor-protocol';
+import type { ComponentConfig, FieldOption, UiComponentProps } from '../../../types/config';
+import './Radio.css';
+import { baseComponentFields, defaultBaseComponent } from '../base';
+import IconSvg from './Radio.svg?react';
+import { Field, Label, RadioGroup, RadioGroupItem } from '@axonivy/ui-components';
+
+type RadioProps = Prettify<Radio>;
+
+const layoutOptions: FieldOption<AlignmentType>[] = [
+  { label: 'horizontal', value: 'lineDirection' },
+  { label: 'vertical', value: 'pageDirection' }
```

Comment:
why lane and pageDirection? I would call them horizontal and vertical...

Reply:
I have called them pageDirection and lineDirection because the PrimeFace component calls them this way:
![grafik](https://github.com/user-attachments/assets/e586ca6f-b937-4dce-b4b4-371da7f6f4cf)

But I guess it doesn't really matter because the labelling is set to horizontal and vertical so the user already has the right names, right?

Reply:
I would not blindly use the primefaces attributes...
If I read the f.json and read orientation="horizontal" it would much clearer to me than layout="lineDirection". Of course we need to translate it later to primefaces... But my opinion.